lib/simplecov/defaults.rb

```diff
@@ -83,14 +83,13 @@
         if coverage_diff > SimpleCov.maximum_coverage_drop # rubocop:disable Metrics/BlockNesting
           $stderr.printf("Coverage has dropped by %.2f%% since the last time (maximum allowed: %.2f%%).\n", coverage_diff, SimpleCov.maximum_coverage_drop)
           @exit_status = SimpleCov::ExitCodes::MAXIMUM_COVERAGE_DROP
+        else
+          SimpleCov::LastRun.write(:result => {:covered_percent => covered_percent})
         end
+      else
+        SimpleCov::LastRun.write(:result => {:covered_percent => covered_percent})
```

Comment:
I think I'd just move the write tot he bottom of the bigger if and keep the `exit_status` check as before I think. But this whole code is ripe for some proper refactoring though :)